Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/Location.hs

```diff
@@ -59,9 +61,84 @@ data LocationAPIEntity = LocationAPIEntity
   }
   deriving (Generic, FromJSON, ToJSON, Show, ToSchema)
 
+data DummyLocationInfo = DummyLocationInfo
+  { dummyId :: Text,
+    lat :: Double,
+    lon :: Double,
+    distance :: Meters,
+    requestedVehicleVariant :: Variant,
+    street :: Maybe Text,
+    door :: Maybe Text,
+    city :: Maybe Text,
+    state :: Maybe Text,
+    country :: Maybe Text,
+    building :: Maybe Text,
+    areaCode :: Maybe Text,
+    area :: Maybe Text,
+    fullAddress :: Maybe Text
+  }
+  deriving (Generic, Show, Read, Eq, PrettyShow, ToJSON, FromJSON, ToSchema)
+
 makeLocationAPIEntity :: Location -> LocationAPIEntity
 makeLocationAPIEntity Location {..} = do
   let LocationAddress {..} = address
   LocationAPIEntity
     { ..
     }
+
+dummyFromLocationData :: DummyLocationInfo
+dummyFromLocationData =
+  DummyLocationInfo
+    { dummyId = "dummyFromLocation",
+      lat = 12.9421783,
+      lon = 77.62205,
+      distance = Meters 1000,
+      requestedVehicleVariant = AUTO_RICKSHAW,
+      door = Just "817",
+      building = Just "20th Main Rd",
+      street = Just "Koramangala 8th Block",
+      area = Just "Koramangala, Koramangala 8th Block, 20th Main Rd",
+      areaCode = Just "560095",
+      city = Just "Bengaluru",
+      state = Just "Karnataka 560095",
+      country = Just "India",
+      fullAddress = Just "817, 20th Main Rd, Koramangala 8th Block, Koramangala, Bengaluru, Karnataka 560095, 560095, India"
+    }
+
+dummyToLocationData :: DummyLocationInfo
+dummyToLocationData =
+  DummyLocationInfo
+    { dummyId = "dummyToLocation",
+      lat = 12.938797,
+      lon = 77.624116,
+      distance = Meters 1000,
+      requestedVehicleVariant = AUTO_RICKSHAW,
+      door = Just "831",
+      building = Just "17th F Main Rd",
+      street = Just "6th Block",
+      area = Just "Koramangala, 6th Block",
+      areaCode = Just "560095",
+      city = Just "Bengaluru",
+      state = Just "Karnataka 560095",
+      country = Just "India",
+      fullAddress = Just "Rohit 17th F Main Rd, 6th Block, Koramangala, Bengaluru, Karnataka 560095, 560095, India"
+    }
+
+testValueForDummyLocationDbData :: String
```

Comment:
Why do we need this?

Reply:
Kept it for checking parsing, will remove it.

Reply:
Fixed.